src/server/getters/get-platform-activity-stats.ts

```diff
@@ -0,0 +1,40 @@
+import * as t from "io-ts";
```

Comment:
Left over since both PRs are so related. I'll fix the inevitable conflicts after the first PR is merged into master.